tests/linux_benchmarks/speccpu2006_benchmark_test.py

```diff
@@ -281,9 +280,6 @@ class DummyVM(object):
   def __init__(self):
     self.num_cpus = 256
 
-  def GetMachineTypeDict(self):
```

Comment:
why remove not rename GetResourceMetadata()?

Reply:
This gets added as part of the default metadata - there's no reason that I can see to add it at the benchmark level